src/api/routes/getArticle.js

```diff
@@ -0,0 +1,10 @@
+import { Router } from 'express';
```

Comment:
can you please rename this router to ```articlesRouter```  so that all of us will use it

Reply:
already done it